tests/README.md

```diff
@@ -15,6 +15,13 @@ $ conda install pytest pytest-cov pytest-mock
 
 ## Running Tests
 
+Before running the tests, install the Open-CE pip package using 
+following command - 
+```shell
+$ python setup.py install
```

Comment:
Would it be better to recommend calling pip directly?
```suggestion
$ pip install -e .
```

Reply:
Yeah. That will also do! I'll change the documentation.